pyphare/pyphare/pharein/restarts.py

```diff
@@ -9,20 +9,24 @@
 
 def validate(sim):
 
-    sim.restart_options["timestamps"] = phare_utilities.np_array_ify(sim.restart_options["timestamps"])
-    init_time = sim.start_time()
-
-    timestamps = sim.restart_options["timestamps"]
-    if np.any(timestamps < init_time):
-        raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be less than simulation.init_time({init_time}))")
-    if np.any(timestamps > sim.final_time):
-        raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be greater than simulation.final_time({sim.final_time}))")
-    if not np.all(np.diff(timestamps) >= 0):
-        raise RuntimeError(f"Error: {clazz}.{key} not in ascending order)")
-    if not np.all(np.abs(timestamps / sim.time_step - np.rint(timestamps/sim.time_step) < 1e-9)):
-        raise RuntimeError(f"Error: {clazz}.{key} is inconsistent with simulation.time_step")
-
-    sim.restart_options["timestamps"] = conserve_existing(sim)
+    if "elapsed_timestamps" in sim.restart_options:
+        sim.restart_options["elapsed_timestamps"] = phare_utilities.np_array_ify(sim.restart_options["elapsed_timestamps"])
+
```

Comment:
there are checks for timestamps that I guess don't make sense for elapsed ones. Maybe there are few (others) that make sense to do here though.
I think we should probably not allow elapsed timestamps that are closer than XX time units (to determine). First, close elapsed time stamps may stall the simulation.. worse timestamps too close to each other may be closer than the time it takes to dump, and the simulation would end up dumping two checkpoints almost simultaneously.

It's hard to define a minimum time interval... I'd think 1 hour is a good start, we may want to have a minimum time that depends on how much data is to write (roughly estimated from the init I guess) in future versions of the feature.

Reply:
this is done